### PR TITLE
Namespace added to interpolation constants

### PR DESCRIPTION
**Pull request by cossorzano**

### CODE REVIEW

**Comment by DStrelak** [comment]
Please follow the naming convention for PR.
For successful build when you do some changes affecting both Xmipp and XmippCore, branches on Xmipp(Core) have to have the same name.

File: core/transformations_defines.h [hunk 1]
```diff
@@ -26,16 +26,16 @@
...
-#define BSPLINE3 3
-#define BSPLINE4 4
 
-#define IS_INV true
-#define IS_NOT_INV false
-#define DONT_WRAP false
-#define WRAP true
+namespace xmipp_transformation
+{
+	enum XmippInterpolation {NEAREST=0, LINEAR=1, BSPLINE2=2, BSPLINE3=3, BSPLINE4=4}; // Interpolation type
```
Comment (DStrelak) [inline core/transformations_defines.h:32]: please use enum class (see e.g. [this link](https://stackoverflow.com/a/18335862))

Comment (cossorzano) [reply core/transformations_defines.h:32]: If I use class, the constants would expand to xmipp_transformation::XmippInterpolation::NEAREST. Without the class, we remove at least one level, and there is no ambiguity, anyway.

Comment (cossorzano) [reply core/transformations_defines.h:32]: I think it follows the name convention: both branches are called co_libsvm

Comment (DStrelak) [reply core/transformations_defines.h:32]: enum class are preferred because they also cannot be cast to e.g. int
Also, you can skip the namespace then

**Review by albertmena — APPROVED** [review]

**Pull request merged**